scripts/dev/node1.sh

```diff
@@ -13,7 +13,4 @@ packages/cli/bin/lodestar dev \
   --rest.namespace '*' \
   --metrics \
   --logLevel debug \
-  --eth1 false \
-  --network.requestCountPeerLimit 1000000 \
-  --network.blockCountTotalLimit 1000000 \
-  --network.blockCountPeerLimit 1000000
```

Comment:
Why remove the rate limit changes?

Reply:
No specific reason other than simplifying the scripts. Since those were added in https://github.com/ChainSafe/lodestar/pull/3858, I just assumed those parameters were specifically used to test this PR.

Reply:
Those set the reqresp limits close to infinity, otherwise dev nodes with ban each other very quickly when syncing

Reply:
Those seem to be deprecated and unused
https://github.com/ChainSafe/lodestar/blob/0f0f533e1ac6163082eb0b7baa7499c4e02ae568/packages/cli/src/options/beaconNodeOptions/network.ts#L30-L31


This PR https://github.com/ChainSafe/lodestar/pull/4828 added a new flag
https://github.com/ChainSafe/lodestar/blob/0f0f533e1ac6163082eb0b7baa7499c4e02ae568/packages/cli/src/options/beaconNodeOptions/network.ts#L250

To disable rate limiting now would have to set `--network.rateLimitMultiplier 0`

Reply:
> Haven't seen that so far, I guess the nodes are instantly synced since we have --reset flag and always start from an empty db.

You need to start external nodes after a while such that they need to sync a sufficient number of slots (thousands)

Reply:
Was not aware of that but makes sense, shall we add `--network.rateLimitMultiplier 0` to dev scripts?

It seems to be disabled during testing as well
https://github.com/ChainSafe/lodestar/blob/0f0f533e1ac6163082eb0b7baa7499c4e02ae568/packages/beacon-node/test/utils/network.ts#L127-L128